website/source/docs/connect/security.html.md

```diff
@@ -12,8 +12,10 @@ description: |-
 
 Connect enables secure service-to-service communication over mutual TLS. This
 provides both in-transit data encryption as well as authorization. This page
-will document how to secure Connect. For a full security model reference,
-see the dedicated [Consul security model](/docs/internals/security.html) page.
+will document how to secure Connect. To try Connect locally, complete the
+[Getting Started guide](https://learn.hashicorp.com/consul/getting-started/connect) or for a full security model reference,
+see the dedicated [Consul security model](/docs/internals/security.html) page. 
```

Comment:
Probably should also link to the connect in production guide which has a lot of security info https://learn.hashicorp.com/consul/developer-mesh/connect-production